### PR TITLE
Water-2746 Two Part Tariff integration tests

### PR DESCRIPTION
**Pull request by Stephan-Hough**
feat/water-2746
-- 2 new scenarios added to the integration test pack
-- Two Part Tariff Summer and Winter bill runs
-- Annual Bill run fixed
-- Config changed to make the integration tests work with Bull MQ

### CODE REVIEW

**Review by james-siteclick — CHANGES_REQUESTED** [review]
CI seems to be failing

File: integration-tests/billing/services/data/return-lines.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+exports.rl1 = {
+  quantity: 10000,
+  startDate: '2018-10-01',
+  endDate:	'2019-10-31',
+  timePeriod: 'month',
+  metadata: 'metadata'
```
[the remark this answers is not in the document]
Comment (Stephan-Hough) [reply integration-tests/billing/services/data/return-lines.js:6]: I believe the meta data is not required. I ran the tests with and without meta data and in both cases the tests passed so I decided to take it out. However, the data is there in a previous commit if this needs to be explored further.

File: config.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ const isProduction = ['production'].includes(process.env.NODE_ENV);
 const isProductionLike = ['production', 'preprod'].includes(process.env.NODE_ENV);
 const crmUri = process.env.CRM_URI || 'http://127.0.0.1:8002/crm/1.0';
 const isLocal = process.env.NODE_ENV === 'local';
-const isTravis = process.env.TRAVIS;
+const isTravis = process.env.TRAVIS || process.env.NODE_ENV === 'test';
```
[the remark this answers is not in the document]
Comment (Stephan-Hough) [reply config.js:10]: Yeah I wasn't sure about this. It's something that Tim suggested I add in. When I was investigating the test failing to execute the with the introduction of BullMQ i believe it was only the lazy db connection that was causing issues.

[the remark this answers is not in the document]
Comment (timtamimi) [reply config.js]: An alternative might be to use `process.env.TEST_MODE`

**Comment by timtamimi** [comment]
Converted to draft pending further tests

**Comment by Stephan-Hough** [comment]
forked and closed in a different pull request https://github.com/DEFRA/water-abstraction-service/pull/1266

**Pull request closed without merging**